### DISCUSSION TITLE
Merge all with group:all

### DISCUSSION THREAD

**Discussion by Asaf-Federman** [renovatebot/renovate#27545] [Request Help]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

Latest

### If you're self-hosting Renovate, select which platform you are using.

Bitbucket Server

### What is your question?

I want to auto merge all the docker images within a specific repository, but I got group:all preset enabled. 
In the logs I noticed that auto merge is disabled although automerge is set to true in the config

        packageRules: [{
            automerge: true,
            automergeType: "branch",
            matchDatasources: ["docker"]
        }]
        
        
In the logs, it shows the following:
PR is not configured for automerge (repository=<repository_name>, branch=<branch_name>)

### Logs (if relevant)

<details><summary>Logs</summary>

```

Copy/paste the relevant log(s) here, between the starting and ending backticks

```

</details>

**Comment by rarkins** [comment]
Automerge will only be true for a branch/PR if every update in the PR is configured with automerge=true

**Reply by Asaf-Federman** [reply]
So I guess you can't create two branches: one that contains all the auto merge packages, and one that contains all the rest, right?

**Reply by rarkins** [reply]
Yes, you can. But you'll need to write package rules to tell renovate what you want

**Reply by Asaf-Federman** [reply]
Can you give me an example of such package rules?

**Reply by rarkins** [reply]
Yes see examples in https://docs.renovatebot.com/configuration-options/ in the packageRules section

**Reply by Asaf-Federman** [reply]
I'm getting the following messages in the log:

         {
           "automerge": true,
           "baseBranch": "master",
           "isModified": false,
           "isPristine": true
         },

           "result": "done",
           "prBlockedBy": "BranchAutomerge",

Nothing was committed to master, no PR was created, and I only see that a branch was created.
Can you please assist me in figuring out what's the issue and how to resolve it?

**Reply by rarkins** [reply]
Please copy/paste in the config you attempted any time you're asking for help. But based on the limited log above "prBlockedBy": "BranchAutomerge" indicates that a branch exists, and the PR doesn't exist because you've asked Renovate to do a branch automerge instead of PR automerge. Renovate will try branch automerging for up to 24 hours before giving up and creating a PR.
